tests/yast2_gui/yast2_firewall_libyui.pm

```diff
@@ -0,0 +1,140 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: yast2-firewall yast2-http-server apache2 apache2-prefork firewalld
```

Comment:
Package line, we don't use it. The description I think it is borrowed from old test case.